### PR TITLE
add module to RuntimeGeneratedFunctions call

### PR DESCRIPTION
**Pull request by xtalax**
Again I think my formatter is broken as it keeps trying to format your stuff, don't know what's going on here

### CODE REVIEW

**Review by ChrisRackauckas — APPROVED** [review]

File: src/systems/pde/pdesystem.jl [hunk 1]
```diff
@@ -108,16 +108,16 @@ struct PDESystem <: ModelingToolkit.AbstractMultivariateSystem
...
-                end
-            end
-        end
+			if isnothing(analytic_func)
+				analytic_func = map(analytic) do eq
+					args = arguments(eq.lhs)
+					p = ps isa SciMLBase.NullParameters ? [] : map(a -> a.first, ps)
+					args = vcat(DestructuredArgs(p), args)
+					ex = Func(args, [], eq.rhs) |> toexpr
+					eq.lhs => drop_expr(RuntimeGeneratedFunction(@__MODULE__, @__MODULE__, ex))
```
Comment (YingboMa) [inline src/systems/pde/pdesystem.jl:117]: This should not hardcoded. Also, I think only the first argument is necessary for correct precompilation caching.

Comment (xtalax) [reply src/systems/pde/pdesystem.jl:117]: This actually doesn't fix the bug I have, I may need to pass down @__MODULE__ from PDESysLibrary

**Comment by YingboMa** [comment]
How do you run your formatter? Also, do you have a forced CRLF setting if you are using Windows?

File: src/systems/pde/pdesystem.jl (hunk 1 above)
Comment (YingboMa) [reply src/systems/pde/pdesystem.jl:117]: You can use the `@RuntimeGeneratedFunction ` macro which only needs two.

**Comment by xtalax** [comment]
It appears to run automatically, I am on MacOS

**Comment by YingboMa** [comment]
Have you tried to run it in a vanilla Julia REPL that you start from a terminal?

**Pull request merged**